transmart-data/ddl/postgres/deapp/de_subject_sample_mapping.sql

```diff
@@ -47,3 +47,15 @@ CREATE INDEX de_subject_smpl_mpng_idx2 ON de_subject_sample_mapping USING btree
 --
 CREATE INDEX idx_de_subj_smpl_trial_ccode ON de_subject_sample_mapping USING btree (trial_name, concept_code);
 
+--
+-- add documentation
+--
+COMMENT ON TABLE deapp.de_subject_sample_mapping IS 'Table links patients to Highdim samples';
```

Comment:
Suggestion:
Table to store information about assays, which are associated with high dimensional observations. Assays are typically associated with a patient. There can be many assays per patient, which may be distinguised by sample code, tissue type, platform, etc.